RELEASE-NOTES.txt

```diff
@@ -12,6 +12,7 @@
 - Added indication that publish url has been copied [#1743](https://github.com/Automattic/simplenote-electron/pull/1743)
 - Disallow partial emails and user's own email from being adding to collaborators email field [#1735](https://github.com/Automattic/simplenote-electron/pull/1735)
 - Fixed keyboard shortcut to toggle markdown preview [#1788](https://github.com/Automattic/simplenote-electron/pull/1788)
+- Fixed an issue where typing a comma in the tag input would insert an empty tag [#1798](https://github.com/Automattic/simplenote-electron/pull/1798)
```

Comment:
This change feels more like a change than a fix, even though it does fix. "Insert a tag when pressing comma" or something like that seems to capture the customer-impact of the change.

Reply:
Well, previously it inserted a tag when pressing comma, it just also allowed empty strings and failed to clean up the input field after doing so.